components/WS/src/HttpServerHandler.h

```diff
@@ -0,0 +1,231 @@
+#ifndef HTTP_SERVER_HANDLER_H_
+#define HTTP_SERVER_HANDLER_H_
+
+#include "HttpRequest.h"
+#include "HttpResponse.h"
+#include "WiFi.h"
+#include <Battery.h>
+#include <FuelGauge.h>
+#include <iostream>
+#include <iterator>
+#include <map>
+#include <sstream>
+#include <stdint.h>
+/**
```

Comment:
```suggestion

/**
```

Separate comment from includes.